lib/gpx.js

```diff
@@ -191,6 +191,12 @@ export function* gpxGen(doc) {
 export function gpx(doc) {
   return {
     type: "FeatureCollection",
-    features: Array.from(gpxGen(doc))
+    features: Array.from(gpxGen(doc)),
+    properties: gpxProperties(doc),
```

Comment:
As discussed in the PR, this should probably be a function rather than adding properties: {} to a FeatureCollection object, because the latter is unspecified.